.github/dependabot.yml

```diff
@@ -22,6 +22,30 @@ updates:
       interval: daily
     target-branch: dev
 
+#  - package-ecosystem: terraform
+#    directory: '/terraform-iac/dev/app'
+#    schedule:
+#      interval: daily
+#    target-branch: dev
+
+#  - package-ecosystem: terraform
+#    directory: '/terraform-iac/dev/setup'
+#    schedule:
+#      interval: daily
+#    target-branch: dev
```

Comment:
I don't love the idea of getting 8 PRs whenever we need to update the AWS provider, but I can't think of a way around that until https://github.com/dependabot/dependabot-core/issues/2178 is added.